tests/mobile/regression/youtube.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect, Locator } from '@playwright/test';
+import { HvMobile } from '../../page_objects/mobile_hv';
+
+/**
+ * The bug here was every time you re-open the shared youtube videos popup,
+ * the text "No shared movies found" was appended to the DOM each time.
+ * It should only appear once.
+ *
+ * Test steps:
+ *   1. Open the youtube dialog
+ *   2. Expect that the text appears once
+ *   3. Close and re-open the youtube dialog
+ *   4. Expect that the text still appears once
+ */
+test('[Mobile] "No shared movies found" should not be duplicated', async ({ page }) => {
+  let mobile = new HvMobile(page);
+  await mobile.Load();
+  await mobile.OpenYoutubeVideosDialog();
+  await expect(page.getByText("No shared movies found.")).toHaveCount(1);
```

Comment:
It would be great if had function like ; 
`mobile.AssertNoYoutubeSharedVideosPresent`
not a blocker of course,

Reply:
I wonder if we can extend `expect` to do `expect(mobile).toDoOurAssertions`

This way the assertions aren't hidden in our class.

Reply:
I couldn't get my head around `expect(mobile).toDoOurAssertions` 
My intention with `AssertNoYoutubeSharedVideosPresent` was, if we ever want to rewrite hv, 
We can just rewrite `AssertNoYoutubeSharedVideosPresent` and do not have to change tests.
 But of course we have other dimensions, `mobile, embed, ... ` not fully sure how we should abstract that part, 
 not even sure , we need to ,

Reply:
We can override expect, but it looks like more work than I want to do.
https://playwright.dev/docs/test-assertions

I did move the assertions into a function on mobile though.